src/main/kotlin/com/example/noswimmingserver/domain/common_user/domain/User.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.noswimmingserver.domain.common_user.domain
```

Comment:
🚫 **[ktlint]** <package-name> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Package name must not contain underscore (cannot be auto-corrected)